package.json

```diff
@@ -1,25 +1,88 @@
 {
-  "name": "pig",
+  "name": "ts",
```

Comment:
why do we have this arbritrary name instead of pacty-parrots or something more descriptive?

Reply:
Absolutely agree! Changing now